src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -893,6 +901,11 @@ public void init(final boolean forceNewConnection) {
                 final String checkName = (String) checkConfig.get("check_name");
                 for (Map<String, Object> configInstance : configInstances) {
                     log.info("Instantiating instance for: " + checkName);
+                    final String instanceName = (String) configInstance.get("name");
```

Comment:
this could return `null`, what would happen to your code if it does?

Reply:
made new commit to add a `null` check